02-express-tutorial/authorize.js

```diff
@@ -0,0 +1,13 @@
+const authorize = (req, res, next) => {
+    const {user} = req.query;
+    if(user === 'john'){
+        req.user = {name: 'john', id: 3}    
+        next();
+    }
+    else{
+        res.status(401).send('Unauthorized');
+    }
+    console.log("authorize");
+    next();
```

Comment:
Requires change:
In the `else` condition, once you invoke `res.status(401).send("Unauthorized");` you don't need  invoke `next()`.